grouper/ctl/user.py

```diff
@@ -2,7 +2,15 @@
 
 from grouper import public_key
 from grouper.ctl.util import ensure_valid_username, make_session
-from grouper.models import AuditLog, User
+from grouper.models import AuditLog, User, get_all_users
```

Comment:
i believe we're inconsistent. :/ fwiw, i've started doing whatever `:sort` does in VI. ;)